x/profile/types/types.go

```diff
@@ -39,13 +39,12 @@ func IsValidAvatar(str string) bool {
 
 // Public profile data
 type Public struct {
-	FirstName    string `json:"firstName"`
-	LastName     string `json:"lastName"`
-	Avatar       string `json:"avatar"`
-	Bio          string `json:"bio"`
-	Gender       Gender `json:"gender"`
-	Birthday     string `json:"birthday"`
-	RegisteredAt int64  `json:"registeredAt"`
```

Comment:
The Frontend uses this field as a starting point in the user's PDV graph. How will it handle with this change?

Reply:
Theseus knows when, and for posts from Genesis they should use something else